<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_CryMaya_Align_Tool</class>
 <widget class="QDialog" name="Dialog_CryMaya_Align_Tool">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CryMaya Align Tool</string>
  </property>
  <widget class="QGroupBox" name="grp_alignPosition">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Align Position (World):</string>
   </property>
   <widget class="QCheckBox" name="chk_pos_xPos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X Position</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_pos_yPos">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Position</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_pos_zPos">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Z Position</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grp_sourceObject">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Source Object:</string>
    </property>
    <widget class="QRadioButton" name="rb_pos_src_min">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimum</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_pos_src_center">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Center</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_pos_src_pivot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Pivot Point</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_pos_src_max">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Maximum</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grp_targetObject">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Target Object:</string>
    </property>
    <widget class="QRadioButton" name="rb_pos_tar_min">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimum</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_pos_tar_center">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Center</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_pos_tar_pivot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Pivot Point</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_pos_tar_max">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Maximum</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_alignOrientation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Align Orientation (Local):</string>
   </property>
   <widget class="QCheckBox" name="chk_orient_zAxis">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Z Axis</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_orient_yAxis">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Axis</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_orient_xAxis">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X Axis</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_matchScale">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Match Scale:</string>
   </property>
   <widget class="QCheckBox" name="chk_scale_zAxis">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Z Axis</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_scale_yAxis">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Axis</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_scale_xAxis">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X Axis</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_apply">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
